### ISSUE TITLE
[Feature] i bought a schlagen gt wich is 500K plus but the car only goes 100mph max so is it able to get this a bit higher since it's a sport/racing car

### ISSUE THREAD

**Issue by Tattguynick** [FrenzyRoleplay/server-issues#606]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ripsupreme** [comment]
YES PLEASE I AGREE, I also bought it, upgraded engine, turbo and transmission and only goes 115
